repos/c2sm/packages/py-tabulate/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from spack.pkg.builtin.py_tabulate import PyTabulate as SpackPyTabulate
+
+
+class PyTabulate(SpackPyTabulate):
```

Comment:
same here

Reply:
v0.8.10 is a min requirement by gt4py. I just noticed it during this PR, so let's keep it?